srcalc/src/main/java/gov/va/med/srcalc/web/controller/admin/EditRuleController.java

```diff
@@ -41,15 +41,31 @@ public EditRuleController(final AdminService adminService)
         super(adminService);
     }
     
+    /**
+     * Presents a form to edit the specified Rule.
+     * @param ruleId the surrogate key of the rule to edit
+     * @throws InvalidIdentifierException if the specified rule does not exist
+     */
     @RequestMapping(method = RequestMethod.GET)
     public ModelAndView displayForm(@PathVariable("ruleId") final int ruleId)
             throws InvalidIdentifierException
     {
         final EditExistingRule editRule = new EditExistingRule(
                 getAdminService().getRuleById(ruleId));
+        // displayForm() should never throw an InvalidIdentifierException here because the
+        // rule should always reference valid variables at this point.
```

Comment:
It was a bit vague to me when I read the comment, I think you should specify that calling displayForm(EditRule) should not throw the exception.